omnibus/config/software/datadog-cf-finalize.rb

```diff
@@ -29,8 +29,12 @@
 
             copy "#{cf_source_root}/agent/agent.exe", "#{cf_bin_root_bin}"
             copy "#{cf_source_root}/agent/libdatadog-agent-three.dll", "#{cf_bin_root_bin}"
-            copy "#{cf_source_root}/agent/install-cmd.exe", "#{cf_bin_root_bin}/agent"
-            copy "#{cf_source_root}/agent/uninstall-cmd.exe", "#{cf_bin_root_bin}/agent"
+
+            unless windows_arch_i386?
+              copy "#{cf_source_root}/agent/install-cmd.exe", "#{cf_bin_root_bin}/agent"
+              copy "#{cf_source_root}/agent/uninstall-cmd.exe", "#{cf_bin_root_bin}/agent"
+            end
```

Comment:
Disabling this since the executables are not generated.
https://github.com/DataDog/datadog-agent/blob/52e3c19c4cf02a20be958f8fe9f2d02a09a942eb/omnibus/config/software/datadog-agent.rb#L94-L97

Re-tested and this should be all